pack/repo.go

```diff
@@ -73,3 +74,11 @@ func (r packMgoRepo) FindAll() ([]Pack, error) {
 
 	return ps, err
 }
+
+func (r packMgoRepo) RemoveAllOlderThan(date time.Time) (info *mgo.ChangeInfo, err error) {
+
+	s := mongo.GetSession()
+	defer s.Close()
+
+	return s.DB(mongo.DbName).C(mongo.PackCollectionId).RemoveAll(bson.M{"lastSeen": bson.M{"$lt": date}})
```

Comment:
I'm not sure about leaking internal details outside the repo. Should we only return the number of items removed instead of this mongo object. I think we are only using one field of it.

It will make your test easier too.

Reply:
Totally agree - I've changed this